scripts/build.js

```diff
@@ -116,8 +116,8 @@ function buildFile(file, silent) {
             plugin[0],
             Object.assign({}, plugin[1], {
               lazy: string =>
-                // we want to lazyload all non-local modules plus `importEsm` - the latter to avoid syntax errors. Set to just `true` when we drop support for node 8
-                !string.startsWith('./') || string === './importEsm',
+                // we want to lazyload all non-local modules - this is buggy though as it doesn't pick up `../` etc. We should consider just `: true` which works correctly or even () => true which would also transform any local imports
```

Comment:
my b - https://github.com/facebook/jest/commit/e818dca70b1d314714a98e7ad79e8b73f1af9ec2#diff-a992dd64a8c626fedc95d613f43602b91f6dc6339893afaf405dc05fa13c9ceaR157

Reply:
I'll change this to `: true`, just wanna verify CI is happy with this version first